src/components/App.js

```diff
@@ -22,12 +23,45 @@ class App extends React.Component {
     this.setState({currentMovie: null})
   }
 
+  errorHandling = () => {
```

Comment:
maybe instead of just setting the error key to true, it could take the error message from the catch and store that? So that we could surface it to the user?